### DISCUSSION TITLE
Make processed read output optional

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#21] [General]
Right now reads from cutadapt are stored in the results folder. My guess is that many users probably do not need that. 

We could make it optional using something like `--store_reads` - which is set to false by default. That would then go into the publish block:

```NEXTFLOW
publishDir = [
            path: { "${params.outdir}/cutadapt" },
            mode: params.publish_dir_mode,
            enabled: params.store_reads
        ] 
```

Would that work for you?

**Comment by gregdenay** [comment]
Sounds good.
We should probably mention the use of `nextflow clean` in the doc to avoid users that are not familiar with nextflow´s workings from bloating their disk.
